pkg/http/api.go

```diff
@@ -224,6 +233,32 @@ func (a *api) MarkStatusAsOutboundReady() {
 	a.outboundReadyStatus = true
 }
 
+// Workflow Component: Component specified in yaml (temporal, etc..)
+// Workflow Type: Name of the workflow to run (function name)
+// Instance ID: Identifier of the specific run
+func (a *api) constructWorkflowEndpoints() []Endpoint {
+	return []Endpoint{
+		{
+			Methods: []string{fasthttp.MethodGet},
+			Route:   "workflows/{workflowComponent}/{workflowType}/{instanceID}",
```

Comment:
Per my other comment, we should consider removing `/{workflowType}` from this route since it's not being used.
```suggestion
			Route:   "workflows/{workflowComponent}/{instanceID}",
```